main.go

```diff
@@ -8,7 +8,6 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth/exec"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
-	_ "k8s.io/client-go/plugin/pkg/client/auth/openstack"
```

Comment:
@tbugfinder Hey! Thanks for your contribution. This looks perfect except this change.

Did you remove this due to that the OpenStack auth plugin isn't included in the upstream, by any chance? If so, do we have an alternative? Without an alternative, we might end up dropping support for OpenStack users.

Reply:
Yes, it was removed in upstream.

https://github.com/kubernetes/client-go/commit/98e81a77843e9ef2ad5c98a6919e450028610631

Reply:
Thanks! So we'd need to add the external cloud provider explicitly to bring back the removed functionality here, right?

Context: https://github.com/kubernetes/kubernetes/pull/67782

Reply:
Ah okay so they moved the client auth to the exec credentials and we already import `https://github.com/kubernetes/client-go/tree/master/plugin/pkg/client/auth/exec` above so... this looks good overall!